scripts/standalone.mk

```diff
@@ -153,8 +153,13 @@ RISCV_CFLAGS    += -march=$(RISCV_ARCH) -mabi=$(RISCV_ABI) -mcmodel=$(RISCV_CMOD
 RISCV_CXXFLAGS  += -march=$(RISCV_ARCH) -mabi=$(RISCV_ABI) -mcmodel=$(RISCV_CMODEL)
 RISCV_ASFLAGS   += -march=$(RISCV_ARCH) -mabi=$(RISCV_ABI) -mcmodel=$(RISCV_CMODEL)
 # Prune unused functions and data
+ifeq ($(PROGRAM),dhrystone)
+RISCV_CFLAGS   += -fno-function-sections -fno-data-sections
+RISCV_CXXFLAGS += -fno-function-sections -fno-data-sections
```

Comment:
This is going to break our current/recently-added method of reducing ITIM consumption by linking only the right function sections into the ITIM.

Reply:
> This is going to break our current/recently-added method of reducing ITIM consumption by linking only the right function sections into the ITIM.

Ah yes I forgot about that. We need to address that going forward. For now, I will limit this to on 8-series then.

Reply:
As long as the U8 doesn't need code in ITIM, this at least won't break non 8-series targets.

Reply:
8-series does not support TIMs of any sort (neither I nor D).